tests/platforms/kubernetes/appmanager.go

```diff
@@ -12,101 +12,93 @@ import (
 
 	"istio.io/pkg/log"
 
+	"github.com/dapr/dapr/tests/utils"
 	appsv1 "k8s.io/api/apps/v1"
 	apiv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
 const (
+	// MiniKubeIPEnvVar is the environment variable name which will have Minikube node IP
+	MiniKubeIPEnvVar = "MINIKUBE_IP"
```

Comment:
do the utils assume Minikube usage? What docker for desktop k8s is used?

Reply:
This is the initial commit of e2e test utils. We're targeting to only minikube for local development experience in this iteration. kind, k8s in docker desktop, etc will be next.

Reply:
@mmisztal1980 If you are interested in the spec and the progress of e2e test, please see the below links
* https://github.com/dapr/dapr/issues/335#issuecomment-548217582
* https://github.com/orgs/dapr/projects/9#card-28802684

Reply:
I am, cheers!